ios/sound-send/sound-send/ViewController.swift

```diff
@@ -9,17 +9,98 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    //the base url, change it when we have one i guess?
+    let baseurl = "http://www.soundsend.com"
+    //keep track of whether a session is in progress. there shouldn't be a session made if one is in progress
+    var sessioninprogress = false
```

Comment:
Remove this variable and all its uses. It's usually a bad idea to introduce state when we don't need it. I don't imagine a use case where the user is allowed to create a channel if a previous one has not been canceled.